docs/DeveloperGuide.md

```diff
@@ -113,7 +113,7 @@ The `UI` component,
 
 **API** : [`Logic.java`](../src/main/java/seedu/address/logic/Logic.java)
 
-1. `Logic` uses the `Parser` class to parse the user command.
+1. `Logic` uses the `MainParser` class to parse the user command. `MainParser` relies on `Parser` of [Natty by Joe Stelmach](https://github.com/joestelmach/natty) for natural language processing.
 2. This results in a `Command` object which is executed by the `LogicManager`.
 3. The command execution can affect the `Model` (e.g. adding a person) and/or raise events.
```

Comment:
"person here"
btw i'm not being naggy ok ~ since i'm already going through i'll try to mark those that I can find so that u guys have an easier life to do the neccessary changes ~ however, do note to not only change the names but also the explanation if neccessary ~ when we go through your logic codes, we expect it to run in a similar manner as explained